modules/logging-release-notes-5-6-16.adoc

```diff
@@ -0,0 +1,12 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-16_{context}"]
+= Logging 5.6.16
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0695[Logging Bug Fix 5.6.16]
+
+[id="logging-release-notes-5-6-16-bug-fixes"]
+== Bug fixes
+* Before this update, when configured to read a custom S3 Certificate Authority the Loki Operator would not automatically update the configuration when the name of the ConfigMap or the contents changed. With this update, the Loki Operator is watching for changes to the ConfigMap and automatically updates the generated configuration. (link:https://issues.redhat.com/browse/LOG-4967[LOG-4967])
```

Comment:
Reword as: "...the Loki Operator watches for changes to the ConfigMap...

Reply:
This is 1 of 3 releases that are live today, unless this change is necessary per one of the style guides, I'd just as soon leave it as is.